contracts/test/fixture/_pool_tilt.js

```diff
@@ -0,0 +1,142 @@
+        // important for Balancer type pools order of these must match the Balancer pool
+        assetAddressArray: [fixture.reth.address, fixture.weth.address],
+        bptToken: rEthBPT,
+        balancerVault: fixture.balancerVault,
+      },
+    };
+  } else if (
+    !!sfrxETHwstETHrEthBPT &&
+    pool.address == sfrxETHwstETHrEthBPT.address
+  ) {
+    return {
+      config: {
+        platform: "balancer",
+        poolType: "composableStable",
+        poolId: await fixture.balancerSfrxWstRETHStrategy.balancerPoolId(),
+        pool: sfrxETHwstETHrEthBPT,
+        // important for Balancer type pools order of these must match the Balancer pool
+        assetAddressArray: [
+          sfrxETHwstETHrEthBPT.address,
+          addresses.mainnet.wstETH,
+          addresses.mainnet.sfrxETH,
+          addresses.mainnet.rETH,
+        ],
+        bptToken: sfrxETHwstETHrEthBPT,
+        balancerVault: fixture.balancerVault,
+      },
+    };
+  }
+}
+
+async function _fundAttackerOption({
+  asset, // attacking asset
+  assetAmount, // required funded asset amount to perform tilt
+  fixture,
+}) {
+  const attackerEthBalance = await waffle.provider.getBalance(attackerAddress);
+  const assetBalance = await asset.balanceOf(attackerAddress);
+
+  if (
+    attackerEthBalance.gte(ousdUnits("100000")) ||
+    assetBalance.gte(assetAmount)
+  ) {
+    // attacker sufficiently funded
+    return;
+  }
+
+  sAttacker = await impersonateAndFundContract(attackerAddress, "1000000"); // 1m ETH
+
+  if (asset.address == fixture.weth.address) {
+    await mintWETH(fixture.weth, sAttacker, "500000");
+  } else if (asset.address == fixture.sfrxETH.address) {
+    const abi = require("../abi/fraxEthMinter.json");
+    const minter = await ethers.getContractAt(
+      abi,
+      addresses.mainnet.FraxETHMinter
+    );
+    await minter
+      .connect(sAttacker)
+      .submitAndDeposit(attackerAddress, { value: assetAmount });
+  }
+}
+
+/* Generic tilt pool function
+ *
+ */
+async function tiltPool({
+  fixture,
+  /* amount of attacking asset in relation to pool TVL. 100 == 100% of TVL meaning
+   * a pool with 1m TVL will get tilted by 1m of attacking asset.
+   */
+  tiltTvlFactor = 100,
+  attackAsset, // asset used to tilt the pool
+  poolContract,
+}) {
+  const poolConfig = await getPoolConfiguration(fixture, poolContract);
+  pool = await factoryCreatePool(poolConfig.config);
+
+  const tvl = await pool.getTvl();
+  const tiltAmount = tvl
+    .mul(BigNumber.from(tiltTvlFactor))
+    .div(BigNumber.from("100"));
+
+  // fund the attacker
+  await _fundAttackerOption({
+    asset: attackAsset,
+    // fund with some 20% extra funds
+    assetAmount: tiltAmount
+      .mul(BigNumber.from("120"))
+      .div(BigNumber.from("100")),
+    fixture,
+  });
+
+  await pool.tiltPool(tiltAmount, attackAsset, sAttacker);
+}
+
+/* Generic tilt pool function
+ *
+ */
+async function unTiltPool({
+  // eslint-disable-next-line no-unused-vars
+  fixture,
```

Comment:
`fixture` and `poolContract` seem to be unused, so can remove them right?

Reply:
I was thinking of leaving these in since they might be used by curve. On the other hand I see your point, no need to future proof these APIs. Removing them